mongo/tests/test_integration.py

```diff
@@ -33,7 +33,17 @@ def test_integration_mongos(instance_integration, aggregator, check):
 
     _assert_metrics(
         aggregator,
-        ['default', 'custom-queries', 'dbstats', 'indexes-stats', 'collection', 'connection-pool', 'jumbo', 'sessions'],
+        [
+            'count-dbs',
```

Comment:
`default` metrics have been split into two fixture files:
`count-dbs`: with only the metric `mongodb.dbs`
`serverStatus`: for the rest